src/main/java/games/strategy/engine/lobby/server/userDB/DerbyUserController.java

```diff
@@ -156,18 +165,20 @@ public boolean login(final String userName, final String hashedPassword) {
   }
 
   @Override
-  public DbUser getUser(final String userName) {
+  public DbUser getUserByName(final String userName) {
```

Comment:
@ron-murhammer I think the latest two commits will address this concern a bit more - notably I ran across where we printed 'username' to external (to user). I updated that to be 'name' instead of 'username'.   Ideally IMO we'd drop 'username' in favor of 'name' when it is printed to users, and use 'userName' or 'name' internally. @ron-murhammer I think the latest two commits will address this concern a bit more - notably I ran across where we printed 'username' to external (to user). I updated that to be 'name' instead of 'username'.   Ideally IMO we'd drop 'username' in favor of 'name' when it is printed to users, and use 'userName' or 'name' internally.